ega-data-api-dataedge/src/test/java/eu/elixir/ega/ebi/dataedge/service/internal/RemoteFileServiceImplTest.java

```diff
@@ -21,10 +21,10 @@
 import eu.elixir.ega.ebi.commons.shared.dto.MyExternalConfig;
 import eu.elixir.ega.ebi.commons.shared.service.DownloaderLogService;
 import eu.elixir.ega.ebi.commons.shared.service.FileInfoService;
-import eu.elixir.ega.ebi.dataedge.config.InternalErrorException;
-import eu.elixir.ega.ebi.dataedge.config.NoContentException;
-import eu.elixir.ega.ebi.dataedge.config.UnavailableForLegalReasonsException;
 import eu.elixir.ega.ebi.dataedge.dto.*;
+import eu.elixir.ega.ebi.dataedge.exception.InternalErrorException;
+import eu.elixir.ega.ebi.dataedge.exception.NoContentException;
```

Comment:
Again if possible use the options in `ega-data-api-commons` in order to do not duplicate code :)